container-core/src/main/java/com/yahoo/container/core/documentapi/VespaDocumentAccess.java

```diff
@@ -72,14 +64,7 @@ public void shutdown() {
     }
 
     void protectedShutdown() {
-        delegate.updateAndGet(access -> {
-            super.shutdown();
-            shutDown = true;
-            if (access != null)
-                access.shutdown();
-
-            return null;
-        });
+        delegate.close();
```

Comment:
No longer calls `super.shutdown()`, but this does nothing for this setup anyway.